packages/rocketchat-lib/server/functions/createRoom.js

```diff
@@ -1,5 +1,5 @@
 /* globals RocketChat */
-RocketChat.createRoom = function(type, name, owner, members, readOnly, extraData={}) {
+RocketChat.createRoom = function(type, name, owner, members, isAutomatic=false, readOnly, extraData={}) {
```

Comment:
Have you made sure you updated all calls to this function everywhere in the codebase?